src/checkoutservice/main.go

```diff
@@ -130,6 +156,26 @@ func initStackDriverTracing() {
 	log.Warn("could not initialize stackdriver exporter after retrying, giving up")
 }
 
+// Initialize OTel trace provider that exports to Cloud Trace
+func InitTraceProvider() {
```

Comment:
nit: camelCase function doesn't match the rest, maybe prefer initTraceProvider?

Reply:
woops, addressed.